src/Translations/Visitor/FormOptionsVisitor/ChildFormLabelsVisitor.php

```diff
@@ -0,0 +1,146 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\PhpCs\Translations\Visitor\FormOptionsVisitor;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Scalar\String_;
+use PhpParser\NodeVisitor;
+
+/**
+ * Visitor specifically for a single form class.
+ */
+class ChildFormLabelsVisitor implements NodeVisitor
+{
+    private const TRANSLATABLE_KEYS = [
+        "help" => true,
+        "label" => true,
+        "placeholder" => true,
+        "title" => true,
+    ];
+
+    /**
+     * @var string[]
+     */
+    private $messages = [];
+
+    /**
+     * @inheritDoc
+     */
+    public function beforeTraverse (array $nodes)
+    {
+    }
+
+
+    /**
+     * @inheritDoc
+     */
+    public function enterNode (Node $node)
+    {
+        if (!$node instanceof MethodCall)
+        {
+            return;
+        }
+
+        $methodName = (string) $node->name;
+        $variableName = $this->findVariableName($node->var);
+
+        if ("add" !== $methodName || !\in_array($variableName, ["form", "builder"], true))
```

Comment:
Isn't this a bit too fragile to rely on the variable's name? Can't we extract all variables in the current scope and check which one implements certain interfaces, like `FormBuilderInterface` and `FormInterface`?

Reply:
This is a parser, not a static analysis. To have the types we would need to actually run the code... which isn't what static extraction is supposed to do.